<ui version="4.0" >
 <author></author>
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>duplicateAccountNumbers</class>
 <widget class="QWidget" name="duplicateAccountNumbers" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Duplicate Account Numbers</string>
  </property>
   <layout class="QHBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="_changeProfit" >
            <property name="text" >
             <string>Change Profit Center Number:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="XComboBox" name="_company" >
            <property name="type" >
             <enum>XComboBox::Companies</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="_changeCompany" >
            <property name="text" >
             <string>Change Company Number:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="XComboBox" name="_profit" >
            <property name="type" >
             <enum>XComboBox::ProfitCenters</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="XComboBox" name="_sub" >
            <property name="type" >
             <enum>XComboBox::Subaccounts</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QCheckBox" name="_changeSub" >
            <property name="text" >
             <string>Change Sub. Accnt. Number:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="_descripLit" >
          <property name="text" >
           <string>Append to Description:</string>
          </property>
          <property name="buddy" >
           <cstring>_descrip</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_descrip" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="_glAccountNumbersLit" >
          <property name="text" >
           <string>Select Account Numbers to Duplicate:</string>
          </property>
          <property name="buddy" >
           <cstring>_account</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XTreeWidget" name="_account" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="_close" >
        <property name="text" >
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_duplicate" >
        <property name="text" >
         <string>Duplicate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_changeCompany</tabstop>
  <tabstop>_company</tabstop>
  <tabstop>_changeProfit</tabstop>
  <tabstop>_profit</tabstop>
  <tabstop>_changeSub</tabstop>
  <tabstop>_sub</tabstop>
  <tabstop>_descrip</tabstop>
  <tabstop>_account</tabstop>
  <tabstop>_duplicate</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>duplicateAccountNumbers</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>486</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
